source/PlayniteSDK/Models/Game.cs

```diff
@@ -1736,6 +1776,11 @@ public override void CopyDiffTo(object target)
                     tro.PlayCount = PlayCount;
                 }
 
+                if (InstallSize != tro.InstallSize)

```

Comment:
Missing for `LastSizeScanDate`.

Reply:
Done in https://github.com/JosefNemec/Playnite/pull/3017/commits/220778689267ab2f0d2aa9adc43ab38030dcf731